packages/components/src/tree-grid/roving-tab-index-context.ts

```diff
@@ -3,7 +3,15 @@
  */
 import { createContext, useContext } from '@wordpress/element';
 
-const RovingTabIndexContext = createContext();
+const RovingTabIndexContext = createContext<
+	| {
+			lastFocusedElement: Element | undefined;
+			setLastFocusedElement: React.Dispatch<
+				React.SetStateAction< Element | undefined >
+			>;
+	  }
+	| undefined
+>( undefined );
```

Comment:
It would be easy to set up a "safer" default for this context value (for example an empty object `{}`), but a [unit test](https://github.com/WordPress/gutenberg/blob/9353567484d1fdaa22295c9bdc037775a9225d62/packages/components/src/tree-grid/test/cell.tsx#L36) implies that we should actually be throwing if the context is not properly initialized. So I'm leaving this as is (just more explicitly undefined I guess).